projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js

```diff
@@ -118,6 +110,20 @@ export default function ProductManagementInspectorControl( {
 								{ __( 'Add subscription', 'jetpack' ) }
 							</Button>
 						</PanelRow>
+						{ siteSlug && (
+							<>
+								<PanelRow>
+									<ExternalLink href={ `https://wordpress.com/earn/payments/${ siteSlug }` }>
+										{ __( 'Manage your subscriptions.', 'jetpack' ) }
+									</ExternalLink>
+								</PanelRow>
+								<PanelRow>
+									<ExternalLink href="https://wordpress.com/support/wordpress-editor/blocks/payments/#related-fees">
+										{ __( 'Read more about Payments and related fees.', 'jetpack' ) }
+									</ExternalLink>
+								</PanelRow>
+							</>
```

Comment:
Note that by adding these into the "Add plan" panel, we are meddling with some styles:

https://github.com/Automattic/jetpack/blob/fd4b2319b8e0c8aae146fcb3c8d9705d9d3b6502/projects/plugins/jetpack/extensions/shared/components/product-management-controls/style.scss#L29-L35

Most importantly, the submit button is supposed to have `margin-top: 25px` which gets assigned to the "read more" row instead, which is also why they have that wonky spacing in the middle. 🙂

Reply:
This said, I wonder if this is the right place for these links.

It makes sense to have "read more" in the same context of the form, but "manage" seems out of place.
What if we moved it into its own panel above the form?

<img width="277" alt="Screenshot 2022-03-23 at 20 09 32" src="https://user-images.githubusercontent.com/2070010/159786963-52468238-b67f-4fc5-8d10-30853965d41a.png">

Roughly:
```js
<InspectorControls>
  { siteSlug && ( <PanelBody>
    <ExternalLink>Manage your subscriptions</ExternalLink>
  </PanelBody> ) }

  <PanelBody title="Add a new subscription">
    { /* ... */ }
    <PanelRow>
      <ExternalLink>Read more</ExternalLink>
    </PanelRow>
  </PanelBody>
</InspectorControls>
```
```scss
// Remove this:
.product-management-control-inspector__add-plan .components-panel__row:last-child {
	margin-top: 25px;
}

// And add this:
.product-management-control-inspector__add-plan .components-panel__row .components-button {
	margin: 16px 0;
}
```

Reply:
I'll put the "Manage your subscriptions" link inside its own panel. And the "Read more about..." link, I'll place it above the "Add subscription" button so that we don't need to change/update the styles.

Reply:
It now looks like this:
![Uploading Captura de pantalla 2022-03-24 a las 10.30.09.png…]()